### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by liquidand** [bilan/connectlife-api-connector#38] [General]
Can someone help me using this integration?. Is there any complete guide illustrating how to setup the environmet ?

**Comment by bilan** [comment]
Hi,
section "Install in Home Assistant with Supervisor" is meant to be that guide. What is unclear?

**Reply by liquidand** [reply]
My installation is core only. I have allso to study a bit mqtt. Seems not so immediate to use

**Reply by bilan** [reply]
Installation is not as straight forward as with Supervisor but you can make it. Just more readings and trial and errors :)

**Reply by liquidand** [reply]
bilan, what to say? Really thank you, the integration after some try works perfectly :)

**Reply by liquidand** [reply]
@bilan is there any way I can integrate the git code into portainer so tu use the update feature using stack deployment? I would like to avoid manual downloading and building of of the repository
